docs/release-promotion/mobile/howto-rc.md

```diff
@@ -60,34 +60,15 @@ python tools/buildfarm/release/trigger_action.py \
 unset PROMOTE_TASK_ID
 ```
 
-This will show you a task definition and ask if you want to submit it (y/n). If you're ready to ship, choose `y`. The ship action taskId will be near the bottom of the output; this taskId is also used as the task graph ID for the ship graph.
+  * The `taskId` of the action task will be the `taskGroupId` of the next graph.
 
-For example, the last output from `trigger_action.py` will look something like this:
-```O - Result:
-{u'status': {u'workerType': u'gecko-3-decision', u'taskGroupId': u'bjVsVQdfSQWjdq9NTBYySA', u'runs': [{u'scheduled': u'2017-11-21T15:40:38.710Z', u'reasonCreated': u'scheduled', u'state': u'pending', u'runId': 0}], u'expires': u'2018-11-21T15:40:09.109Z', u'retriesLeft': 5, u'state': u'pending', u'schedulerId': u'gecko-level-3', u'deadline': u'2017-11-22T15:40:08.109Z', u'taskId': u'OG1t0QchSj209mV9_3tCHA', u'provisionerId': u'aws-provisioner-v1'}}
-```
-
-We see `u'taskId': u'OG1t0QchSj209mV9_3tCHA'` and know that there should be a task graph with that ID, too.
-
-```sh
-taskcluster group list OG1t0QchSj209mV9_3tCHA --all
-```
-
-You can also see the new 'Action: Release Promotion' task on [tools.taskcluster.net](https://tools.taskcluster.net/groups)
-
-### Resolve push-apk-breakpoint task
-
-In the new ship-rc graph, there will be a task with 'push-apk-breakpoint' in the label. To push the `apk` to Google Play, we must resolve this task.
```

Comment:
nice. one less step I suppose without the breakpoint within the graph?

Reply:
yeah. the breakpoint was retired a good while ago so less confusing docs is good 👍